Libraries/test/Amazon.Lambda.RuntimeSupport.Tests/Amazon.Lambda.RuntimeSupport.UnitTests/LogMessageFormatterTests.cs

```diff
@@ -103,6 +103,40 @@ public void FormatJsonWithStringMessageProperties()
             Assert.Equal("Hello AWS", doc.RootElement.GetProperty("message").GetString());
         }
 
+        [Fact]
+        public void FormatJsonNullMessageTemplate()
+        {
+            var timestamp = DateTime.UtcNow;
+            var formatter = new JsonLogMessageFormatter();
+            var state = new MessageState()
+            {
+                MessageTemplate = null,
+                AwsRequestId = "1234",
+                Level = Helpers.LogLevelLoggerWriter.LogLevel.Warning,
+                TimeStamp = timestamp
+            };
+
+            var json = formatter.FormatMessage(state);
+            var doc = JsonDocument.Parse(json);
+            Assert.Equal(string.Empty, doc.RootElement.GetProperty("message").GetString());
+            Assert.Equal("1234", doc.RootElement.GetProperty("requestId").GetString());
+
+            // Currently the arguments are ignored because they don't exist in the message template but 
+            // having arguments uses a different code path so we need to make sure a NPE doesn't happen.
+            state = new MessageState()
+            {
+                MessageTemplate = null,
+                MessageArguments = new object[] { true },
+                AwsRequestId = "1234",
+                Level = Helpers.LogLevelLoggerWriter.LogLevel.Warning,
+                TimeStamp = timestamp
+            };
+
+            json = formatter.FormatMessage(state);
+            Assert.Equal(string.Empty, doc.RootElement.GetProperty("message").GetString());
+            Assert.Equal("1234", doc.RootElement.GetProperty("requestId").GetString());
```

Comment:
this `doc` here is from the line 120. and then on 135 we do `json = formatter.FormatMessage(state);` and im assuming you want to re-assert on the result of that operation (which we dont)